tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>1. Why is this {base_model.layers[0]: 'default'} instead of a list? I don't see anywhere you use 'default'.</p><p> </p><p>2. I suggest keeping the comments for before. It's hard to piece this code together otherwise (not everyone knows what <code>clone_model</code>  does and a short summary of what's applicable is useful.</p><p> </p><p>3. Remove the print statement. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<ol><li>Changed it into apply_clustering_to_dense function since that illustrates the point.</li><li>Done.</li></ol>